test/oComments.test.js

```diff
@@ -34,15 +35,24 @@ describe("OComments", () => {
 		let rootElement;
 		let scriptElement;
 
-		beforeEach(() => {
+		before((done) => {
+			document.addEventListener('oCommentsReady', () => {
+				rootElement = boilerplate.oCommentsEl;
+				scriptElement = rootElement.querySelector('script');
+				done();
+			});
+
+			fetchMock.mock('https://comments-auth.ft.com/v1/jwt/', {
```

Comment:
In here we could add some tests for different responses to compliment my other comment about error handling in the fetch requests